nimbus-features/reduxIntegrationFeature.yaml

```diff
@@ -0,0 +1,18 @@
+# The configuration for the tabStorageRefactorFeature feature
+features:
+  redux-integration-feature:
+    description: >
+      This feature is for managing the roll out of the Redux integration feature
+    variables:
+      enabled:
+        description: >
+          Enables the feature
+        type: Boolean
+        default: false
+    defaults:
+      - channel: beta
+        value:
+          enabled: false
+      - channel: developer
+        value:
+          enabled: true
```

Comment:
Do we want this enabled `true` already?

Reply:
I will disable in the next PR as there is no blocking issue for this one